docs/source/model_design/vivarium_model_components/risk_factors/risk_correlation/index.rst

```diff
@@ -1,1213 +1,152 @@
+result when two normally-distributed risk factors are correlated
+(have non-zero covariance) and the multiplicative PAF is used.
 
-    0.2772216399840989
-
-
-
-.. code:: python
-
-    # how does this compare to multiplicative approx of paf?
-    ooomp_1 = np.mean(rr_1**df['exp_na'])
-    paf_1 = 1 - 1/ooomp_1
-
-    ooomp_2 = np.mean(rr_2**df['exp_nb'])
-    paf_2 = 1 - 1/ooomp_2
-
-    paf_m_ind = 1 - (1 - paf_1) * (1 - paf_2)
-    paf_m_ind
-
-
-
-
-.. parsed-literal::
-
-    0.2783285974850329
-
-
-
-However, when the exposures are correlated, it seems that the PAFs from the the joint risk approach are higher than the multiplicative approach which returns similar values to the independent exposures.
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-
-.. code:: python
-
-    # What is the value of the joint risk approach?
-    one_over_one_minus_paf = np.mean(rr_1**df['exp_ca'] * rr_2**df['exp_cb'])
-    paf_j_cor = 1 - 1/(one_over_one_minus_paf)
-    paf_j_cor
-
-
-
-
-.. parsed-literal::
-
-    0.3420403329275915
-
-
-
-.. code:: python
-
-    # how does this compare to multiplicative approx of paf?
-    ooomp_1 = np.mean(rr_1**df['exp_ca'])
-    paf_1 = 1 - 1/ooomp_1
-
-    ooomp_2 = np.mean(rr_2**df['exp_cb'])
-    paf_2 = 1 - 1/ooomp_2
-
-    paf_m_cor = 1 - (1 - paf_1) * (1 - paf_2)
-    paf_m_cor
-
-
-
-
-.. parsed-literal::
-
-    0.2890061162964448
-
-
-
-Now, calculate individual incidences for the individuals in our simulated data
-==============================================================================
+.. image:: rate_dotplot.jpg
 
-Our proposal is that we leave the general approach to risk deleted
-incidence the same:
+.. todo::
+  Replace this example with Python, in a form we can easily re-run (e.g. Jupyter notebook) and where
+  the parameters used are documented.
+  The R code (incomplete and untested) is at :download:`correlated_exposures_sim.R`.
 
-.. math::
+Calculation of joint PAFs in presence of correlation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Instead of the multiplicative approach, in the case of correlated risk factors we need to
+directly estimate :math:`E(RR_{r1} * RR_{r2})` using our joint distribution between
+:math:`r1` and :math:`r2`.
+This generally needs to be performed for each combination of age, sex, and location.
 
-   i_0 = \left(1-\text{PAF}\right) \cdot i_{\text{GBD}}
+.. todo::
+  Is this usually/always calculated within the simulation, or is it sometimes done on the research side?
+  Is there shared code or components that implement this on the engineering side?
+  If the latter were true, we could simplify this section to "tell engineering to use the correlation-corrected
+  joint PAF calculation" and put the mathematical details elsewhere.
```

Comment:
I am not sure that we have a standard or desired process on this point. I think talking this over with @rmudambi, perhaps using the CVD sim as a case study could be a good opportunity to develop one, but I think we can leave this as a todo note until then.

Reply:
I agree with this ^^ We are going to encounter this shortly with CVD so can decide then and add here.

Reply:
I don't think this section needs to be "simplified" at all, and I like having the mathematical details here. Where else would we put them?

But yeah, once we figure out if there's a standard process or if we create one with the engineers, then we should add that along with the math.

Reply:
I think if we had a standard way to do it that was entirely within Vivarium (the researcher did not even need to be aware of the PAF calculation method, because it was automatically applied whenever a correlation was introduced), then the math would belong in the Vivarium documentation.

But, it seems like we are a long way from that. Keeping this as-is.